EIPS/eip-1062.md

```diff
@@ -0,0 +1,87 @@
+---
+eip: 1062
+title: Formalize IPFS hash into ENS(Ethereum Name Service) resolver
+author: Phyrex Tsai <phyrex@portal.network>, Portal Network Team
+discussions-to: https://ethereum-magicians.org/t/eip-1062-formalize-ipfs-hash-into-ens-ethereum-name-service-resolver/281
+status: Draft
+type: Standards Track
+category: ERC
+created: 2018-05-02
+---
+
+## Simple Summary
+To specify the mapping protocol between resources stored on IPFS and ENS(Ethereum Naming Service).
+
+## Abstract
+The following standard details the implementation of how to combine the IPFS cryptographic hash unique fingerprint with ENS public resolver. This standard provides a functionality to get and set IPFS online resources to ENS resolver.
+  
+We think that this implementation is not only aim to let more developers and communities to provide more use cases, but also leverage the human-readable features to gain more user adoption accessing decentralized resources. We considered the IPFS ENS resolver mapping standard a cornerstone for building future Web3.0 service.
+
+## Motivation
+To build fully decentralized web service, it’s necessary to have a decentralized file storage system. Here comes the IPFS, for three following advantages :
+- Address large amounts of data, and has unique cryptographic hash for every record.
+- Since IPFS is also based on peer to peer network, it can be really helpful to deliver large amounts of data to users, with safer way and lower the millions of cost for the bandwidth.
+- IPFS stores files in high efficient way via tracking version history for every file, and removing the duplications across the network.
+  
+Those features makes perfect match for integrating into ENS, and these make users can easily access content through ENS, and show up in the normal browser.
+
+
+## Specification
+The condition now is that the IPFS file fingerprint using base58 and in the meantime, the Ethereum using hex encryption. These comes the two restrictions :
```

Comment:
It's "hex encoding", not "hex encryption", and it's not actually the case - Ethereum stores binary data, it just uses hex in API to encode the binary data. Internally it's stored in binary form.